packages/bitlib/bitlib.1.0/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+synopsis: "A library for writing binary files"
+maintainer: "Chris Nevers <christophernevers96@gmail.com>"
+authors: "Chris Nevers <christophernevers96@gmail.com>"
+homepage: "https://github.com/chrisnevers/bitlib"
+bug-reports: "https://github.com/chrisnevers/bitlib/issues"
+depends: [
+  "dune" {build}
+  "alcotest" {with-test}
+]
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest"] {with-test}
```

Comment:
```suggestion
  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
```
It's not "really" mandatory but it might be useful later

Reply:
It's also a bit cleaner to use the `run-test` section instead of putting this line into the `build` section.
For instance, if you don't use dune substitutions:
```
build: ["dune" "build" "-p" name "-j" jobs]
run-test: ["dune" "runtest" "-p" name "-j" jobs]
```